vendor/processors/stripe_processor.py

```diff
@@ -640,61 +604,67 @@ def initialize_products(self, site):
                         'price_id': price_id
                     }
 
+    def check_product_does_exist(self, name, metadata):
```

Comment:
rename method to `does_product_exists` and return True or False. 


Also move the method above the `initialize_products` method as it is used inside that method.